### ISSUE TITLE
[URGENT] kiwix-manage is crashing with newly created ZIM files

### ISSUE THREAD

**Issue by kelson42** [kiwix/kiwix-tools#389]
For example:
```
$ kiwix-manage /tmp/tmp.xml add wikipedia_arz_all_maxi_2020-06.zim

Assertion failed at ../src/cluster.cpp:79
 offset[1905477] <= reader->size().v[1905183]
/usr/local/lib/x86_64-linux-gnu/libzim.so.6(_Z15_on_assert_failIjmEvPKcS1_S1_T_T0_S1_i+0x176) [0x7fee89ea471d]
/usr/local/lib/x86_64-linux-gnu/libzim.so.6(_ZN3zim7Cluster11read_headerIjEENS_8offset_tEv+0x294) [0x7fee89ea3520]
/usr/local/lib/x86_64-linux-gnu/libzim.so.6(_ZN3zim7ClusterC2ESt10shared_ptrIKNS_6ReaderEENS_15CompressionTypeEb+0xd8) [0x7fee89ea2130]
/usr/local/lib/x86_64-linux-gnu/libzim.so.6(_ZN3zim8FileImpl10getClusterENS_15cluster_index_tE+0x22a) [0x7fee89eaebf6]
/usr/local/lib/x86_64-linux-gnu/libzim.so.6(_ZNK3zim7Article14getArticleSizeEv+0x67) [0x7fee89e9f231]
/usr/local/lib/x86_64-linux-gnu/libzim.so.6(_ZNK3zim7Article7getDataEm+0x33) [0x7fee89ea02a9]
/usr/local/lib/x86_64-linux-gnu/libkiwix.so.9(_ZNK5kiwix6Reader20parseCounterMetadataB5cxx11Ev+0x124) [0x7fee8a9340cc]
/usr/local/lib/x86_64-linux-gnu/libkiwix.so.9(_ZNK5kiwix6Reader15getArticleCountEv+0x3c) [0x7fee8a934446]
/usr/local/lib/x86_64-linux-gnu/libkiwix.so.9(_ZN5kiwix4Book6updateERKNS_6ReaderE+0x2db) [0x7fee8a9121a1]
/usr/local/lib/x86_64-linux-gnu/libkiwix.so.9(_ZN5kiwix7Manager16readBookFromPathERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEPNS_4BookE+0x113) [0x7fee8a92a701]
/usr/local/lib/x86_64-linux-gnu/libkiwix.so.9(_ZN5kiwix7Manager23addBookFromPathAndGetIdERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEES8_S8_b+0x7c) [0x7fee8a92a240]
kiwix-manage(+0x4379) [0x563cd187b379]
kiwix-manage(+0x4a7b) [0x563cd187ba7b]
/lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xf3) [0x7fee8a46a0b3]
kiwix-manage(+0x350e) [0x563cd187a50e]

```

**Comment by mgautierfr** [comment]
I confirm the issue, however I don't know what is the problem yet.

**Comment by mgautierfr** [comment]
Fixed with openzim/libzim#350
